### PR TITLE
Release ETH version 1.0.9

### PR DESCRIPTION
**Pull request by skchang0710**

### CODE REVIEW

**Review by iancoolbitx — APPROVED** [review]

**Comment by delbinwen** [comment]
What causes lots of addition to package-lock.json? Do we update the version of npm or try to do `npm i`?

File: packages/coin-eth/src/sign.ts [hunk 1]
```diff
@@ -161,15 +161,9 @@ export const signMessage = async (
...
-
-  let msgBuf;
-  if (Web3.utils.isHex(keccak256Msg)) {
-    msgBuf = Buffer.from(removeHex0x(keccak256Msg), 'hex');
-  } else {
-    msgBuf = Buffer.from(keccak256Msg, 'utf8');
-  }
-
+  const msgHex = handleHex(Web3.utils.toHex(message));
+  console.log('msgHex :', msgHex);
```
Comment (delbinwen) [inline packages/coin-eth/src/sign.ts:165]: could we remove this `console.log`?

Comment (skchang0710) [reply packages/coin-eth/src/sign.ts]: sure and done

**Comment by skchang0710** [comment]
Hi all, I  have recovered the package-lock.json file to the previous version.

**Review by iancoolbitx — APPROVED** [review]

**Review by delbinwen — APPROVED** [review]
ok

**Pull request merged**